dashboard/src/components/Tabs/Tabs.tsx

```diff
@@ -48,6 +53,8 @@ const TabsComponent = ({ defaultTab, tabs }: ITabsComponent): JSX.Element => {
   return (
     <Tabs defaultValue={defaultTab.name} className="w-full">
       <TabsList className="w-full justify-start">{tabsTrigger}</TabsList>
+      <div className="py-6">{FilterListElement}</div>
```

Comment:
I'm not sure it is worth it for us to pass the whole element instead of a list with the filters and call the necessary ui here

Reply:
In terms of processing, it's basically the same thing, we are passing a "reference" in both situations, but we gain more flexibility by passing an element here